core/templates/dev/head/services/id-generation.service.spec.ts

```diff
@@ -34,4 +34,22 @@ describe('IdGenerationService', () => {
     let id2 = idGenerationService.generateNewId();
     expect(id1).not.toEqual(id2);
   });
+
+  it('should generate id with 10 digits when random string is greater than' +
+    ' or equal 10 of length', function() {
```

Comment:
`should generate id with 10 digits when random string is greater than or equal 10 of length` -> `should generate id with 10 digits when random string has length greater than or equal to 10`

Reply:
Done.